### DISCUSSION TITLE
Qiita Blog のモバイル表示時のハンバーガーメニューが個別記事ページで動作しない

### DISCUSSION THREAD

**Discussion by k1350** [increments/qiita-discussions#898] [要望・アイデア・不具合]
Qiita Blog はモバイル表示だと右上にハンバーガーメニューが出ます。
このメニューは記事一覧ページだと正しく動作しますが、個別の記事ページだと動作しません。

記事一覧ページ（動作する）
![スクリーンショット 2024-08-31 135003](https://github.com/user-attachments/assets/14ae17f3-0b6c-4257-857a-2615814a8b8a)

個別の記事ページ（ハンバーガーメニューボタンを押しても何も起こらない）
![スクリーンショット 2024-08-31 135017](https://github.com/user-attachments/assets/b9dd5b25-4931-470a-8d06-3037d5b70a8f)

**Comment by DeguchiHiroki** [comment]
ご報告ありがとうございます。
いただいた情報を元に調査を行います。

ご不便をおかけいたしますが、何卒よろしくお願いいたします 🙇

**Comment by DeguchiHiroki** [comment]
表示するコンテンツがないにもかかわらず、ハンバーガーメニューが表示されていたため、
詳細ページのハンバーガーメニューを非表示にする修正を行いました。

ご報告いただき、ありがとうございました🙏
